src/main/java/core/basesyntax/MainApp.java

```diff
@@ -2,4 +2,19 @@
 
 public class MainApp {
 
+    public static void main(String[] args) {
+        Excavator excavator = new Excavator();
```

Comment:
mistake from the checklist 
https://mate-academy.github.io/jv-program-common-mistakes/java-core/oop/oop